crates/oxc_prettier/src/doc.rs

```diff
@@ -15,13 +15,13 @@ pub enum Doc<'a> {
     Array(Vec<'a, Doc<'a>>),
     /// Increase the level of indentation.
     Indent(Vec<'a, Doc<'a>>),
-    IndentIfBreak(Vec<'a, Doc<'a>>),
+    IndentIfBreak(IndentIfBreak<'a>),
```

Comment:
should these be boxed?

Reply:
It doesn't need to be boxed if the compiler doesn't shout at you.